app/scripts/templates/connect_another_device.mustache

```diff
@@ -0,0 +1,52 @@
+<div id="main-content" class="card">
+  <header class="hidden"><h1>{{#t}}Connect another device{{/t}}</h1></header>
+  <div class="success visible">{{#t}}Email confirmed{{/t}}</div>
+  <section>
+    <div class="graphic graphic-connect-another-device">{{#t}}Success{{/t}}</div>
+
+    {{#canSignIn}}
+      <p>
+        {{#t}}Complete set-up by signing in to this Firefox.{{/t}}
+      </p>
+
+      <form novalidate>
+        <div class="button-row">
+          <a id="signin" class="button" href="{{{ escapedSignInUrl }}}">{{#unsafeTranslate}}Sign in &rarr;{{/unsafeTranslate}}</a>
```

Comment:
This anchor is considered an external link since it starts with `https`, and [all external links cause a metrics flush](https://github.com/mozilla/fxa-content-server/blob/a54eac025f0d33e399310cf8ba6b88dd11ea0016/app/scripts/views/mixins/external-links-mixin.js#L70). Since the [external links mixin is mixed into the BaseView](https://github.com/mozilla/fxa-content-server/blob/a54eac025f0d33e399310cf8ba6b88dd11ea0016/app/scripts/views/base.js#L1001:L1004), all links that start with `http` automatically have this behavior.